lib/src/main/kotlin/com/wire/integrations/jvm/WireAppSdk.kt

```diff
@@ -89,4 +94,8 @@ class WireAppSdk(
 
         IsolatedKoinContext.koinApp.koin.loadModules(listOf(dynamicModule))
     }
+
+    private companion object {
+        const val MIN_CRYPTOGRAPHY_STORAGE_PASSWORD = 32
```

Comment:
Nit-picking here, but 32 is not the "MIN", it is the only allowed size, so it should be CRYPTOGRAPHY_STORAGE_PASSWORD_LENGTH

Reply:
renamed to `CRYPTOGRAPHY_STORAGE_PASSWORD_LENGTH`